docs/getting-started/application/index.rst

```diff
@@ -6,55 +6,90 @@ This guide covers a little more detail about how to add oso to your application.
 
-We highly encourage you to follow along with the code by cloning the example repository
-and trying it out. The code can be found here:
+        Our sample expenses application is built with Flask, but we are not using
+        anything from oso that is unique to Flask, and the same patterns we cover here
+        can be used anywhere.
 
-.. todo:: Insert little GitHub snippet box + actually have a real link
+        We highly encourage you to follow along with the code by cloning the example repository
+        and trying it out. The code can be found here:
 
-:fab:`github` `osohq/oso-flask-tutorial <https://github.com/osohq/oso-flask-tutorial>`_
+        .. todo:: Insert little GitHub snippet box + actually have a real link
+
+        :fab:`github` `osohq/oso-flask-tutorial <https://github.com/osohq/oso-flask-tutorial>`_
+
+    .. group-tab:: Java
+
+        Our sample expenses application is a Maven project built with Spring Boot.
+        We are not using anything from oso that is unique to Spring Boot, and the same patterns we cover here
+        can be used anywhere.
+
+        We highly encourage you to follow along with the code by cloning the example repository
+        and trying it out. The code can be found here:
+
+        .. todo:: Insert little GitHub snippet box + actually have a real link
+
+        .. :fab:`github` `osohq/oso-flask-tutorial <https://github.com/osohq/oso-flask-tutorial>`_
 
 Our expenses application reads from a sqlite database, and has a few simple endpoints for returning
 results. We encourage you to take a look around before continuing!
 
 Running The Example
 ===================
 
-The application has a few requirements, including Flask and, or course, oso.
-We recommend installing these within a virtual environment:
+.. tabs::
+    .. group-tab:: Python
 
-.. code-block:: console
+        The application has a few requirements, including Flask and, or course, oso.
+        We recommend installing these within a virtual environment:
 
-    $ git clone https://github.com/osohq/oso-flask-tutorial/
-    $ cd oso-flask-tutorial/
-    $ python3 -m venv venv
-    $ source venv/bin/activate
-    $ pip3 install -r requirements.txt
-    $ FLASK_ENV=development flask run --extra-files app/authorization.polar
+        .. code-block:: console
 
-The example comes with some example data, which you can load with:
+            $ git clone https://github.com/osohq/oso-flask-tutorial/
+            $ cd oso-flask-tutorial/
+            $ python3 -m venv venv
+            $ source venv/bin/activate
+            $ pip3 install -r requirements.txt
+            $ FLASK_ENV=development flask run --extra-files app/authorization.polar
 
-.. code-block:: console
+    .. group-tab:: Java
+
+        After cloning the example app, make sure to run ``mvn install`` to download the necessary JARs.
 
-    $ sqlite3 expenses.db ".read expenses.sql"
+    The example comes with some example data, which you can load with:
```

Comment:
This needs to be updated for Java to point to the correct path (or move it out of src/main so that it is consistent).